src/xdocs/config_imports.xml

```diff
@@ -983,6 +983,60 @@ public class InputIllegalImport { }
         </source>
 
         <p>
+          In the next examples usage of <code>strategyOnMismatch</code> property is shown.
+          This property defines strategy in a case when no matching rule was found.
+          <code>strategyOnMismatch</code> property is attribute of <code>import-control</code> and <code>subpackage</code> tags.
+          Property can have following values:
+          <ul>
+            <li>delegateToParent (only for <code>subpackage</code>)</li>
```

Comment:
it is hard to understand this "only for ...."  please clearly define what is allowed for import-control and subpackage .